ystia/yorc
# Remove force purge tasks autogeneration

# Pull Request description

## Description of the change

### What I did

* Removed autogeneration of a force purge task in list deployments endpoint
* Depreciated force purge task
* updated enum lib to allow adding comments on generated enum types

### How to verify it

1. deploy one or more app 
2. remove the deployment status key in consul `consul kv delete _yorc/deployments/<depID>/status`
3. list deployments with the yorc cli
4. should see a log in yorc and inconsistent deployment should remain

### Description for the changelog

* Yorc generates forcePurge tasks on list deployments API endpoint ([GH-674](https://github.com/ystia/yorc/issues/674))

## Applicable Issues

* Fixes #674 
* Backported to release/4.0 branch in #676
